src/docs/goose-book/src/glossary.md

```diff
@@ -0,0 +1,10 @@
+# Glossary
+
+## Taskset
+A **taskset** in Goose is akin to what we would normally define as a **test
+case** or a **scenario** in other test tools. It is a class of actions or steps
+that the user would undertake to achieve a specific user journey.
+
+## Task
+A **task** is a single step within a **taskset**, like GET a homepage or POST a
+form.
```

Comment:
Tasks are more arbitrary than this. A [`GooseRequest`](https://docs.rs/goose/latest/goose/goose/struct.GooseRequest.html) maps to an individual request. Whereas a [`GooseTask`](https://docs.rs/goose/latest/goose/goose/struct.GooseTask.html) is a collection of one or more requests that typically would simulate a single workflow (such as logging into a website: which may require loading the page, scraping the form, posting the username, login and form elements, and finally loading the resulting profile page and validating the expected elements exist).